src/main/java/ch/wisv/events/model/Event.java

```diff
@@ -0,0 +1,135 @@
+package ch.wisv.events.model;
+
+import com.google.common.base.MoreObjects;
+
+import javax.persistence.*;
```

Comment:
This is because IntelliJ IDEA collapes to star imports if there are more than 5 classes imported from the same package, and places the individual imports back if not.